ci/dotnet-desktop.yml

```
@@ -109,7 +109,7 @@ jobs:
 
     # Upload the MSIX package: https://github.com/marketplace/actions/upload-a-build-artifact
     - name: Upload build artifacts
-      uses: actions/upload-artifact@v3
+      uses: actions/upload-artifact@v4
```

```suggestion
      uses: actions/upload-artifact@v3
```

Thank you for the update @timheuer! I'm recommending we keep `upload-artifact` at `v3` for the moment, as `v4` is not yet supported on GitHub Enterprise: https://github.com/marketplace/actions/upload-a-build-artifact#v4---whats-new